src/bin/customer/close.rs

```diff
@@ -411,30 +412,32 @@ async fn mutual_close(
         &config,
         &channel_details.address,
     )
+    .with_timeout(5 * config.message_timeout)
     .await
-    .context("zkAbacus close failed.")?;
+    .context("Mutual close timed out while preparing to close")?
```

Comment:
Just for my understanding, how does this exactly work, why does it uses the first message on timeout and the second one on a fail?